src/main/java/org/junit/runners/parameterized/BlockJUnit4ClassRunnerWithParameters.java

```diff
@@ -73,6 +73,10 @@ private Object createTestUsingFieldInjection() throws Exception {
             int index = annotation.value();
             try {
                 field.set(testClassInstance, parameters[index]);
+            } catch (IllegalAccessException e) {
+                throw new Exception("Cannot set parameter '" + field.getName()
```

Comment:
Are you sure we shouldn't set the cause of this exception?

Also, throwing ``Exception` is rare. Perhaps we should throw a new `IllegalAccessException`